<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_fraeser_bogen</class>
 <widget class="QDialog" name="Dialog_fraeser_bogen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gebogene Fräsbahn</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="2">
     <widget class="QLineEdit" name="lineEdit_radius"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_ende_x">
      <property name="text">
       <string>Endpunkt in X</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_ende_x_var">
      <property name="text">
       <string>*[X]</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineEdit_ende_x"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_ende_y">
      <property name="text">
       <string>Endpunkt in Y</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_ende_y_var">
      <property name="text">
       <string>*[Y]</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEdit_ende_y"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_ende_z">
      <property name="text">
       <string>Endpunkt in Z</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_ende_z_var">
      <property name="text">
       <string>*[Z]</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="lineEdit_ende_z"/>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_radius_var">
      <property name="text">
       <string>[RAD]</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_radius">
      <property name="text">
       <string>Radius</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_bezeichnung">
      <property name="text">
       <string>Bezeichnung</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_bezeichnung_var">
      <property name="text">
       <string>[BEZ]</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineEdit_bezeichnung"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_ausfbed">
      <property name="text">
       <string>Ausführbedinging</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_ausfbed_var">
      <property name="text">
       <string>[AFB]</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEdit_ausfbed"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>431</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_abbrechen">
      <property name="text">
       <string>Abbrechen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_speichern">
      <property name="text">
       <string>Speichern</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>221</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton_im_UZS">
      <property name="text">
       <string>in Uhrzeigersinn (G02)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_gegen_UZS">
      <property name="text">
       <string>gegen den Uhrzeigersinn (G03)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_ende_x</tabstop>
  <tabstop>lineEdit_ende_y</tabstop>
  <tabstop>lineEdit_ende_z</tabstop>
  <tabstop>lineEdit_radius</tabstop>
  <tabstop>radioButton_im_UZS</tabstop>
  <tabstop>radioButton_gegen_UZS</tabstop>
  <tabstop>lineEdit_bezeichnung</tabstop>
  <tabstop>lineEdit_ausfbed</tabstop>
  <tabstop>pushButton_abbrechen</tabstop>
  <tabstop>pushButton_speichern</tabstop>
  <tabstop>pushButton_ok</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
